submissions/tests/test_models.py

```diff
@@ -166,3 +170,52 @@ def test_highest_score_hidden(self):
         highest = ScoreSummary.objects.get(student_item=item).highest
         self.assertEqual(highest.points_earned, 1)
         self.assertEqual(highest.points_possible, 2)
+
+
+class TestTeamSubmission(TestCase):
+    """
+    Test the TeamSubmission class
+    """
+    def test_create_team_submission(self):
+        user = TestTeamSubmission.create_user('user1')
+        ts = TeamSubmission.objects.create(
```

Comment:
I'd recommend doing your object creation in a `setUpTestData` method: https://docs.djangoproject.com/en/1.11/topics/testing/tools/#django.test.TestCase.setUpTestData

That way, any objects you create will be rolled back once this test class is finished, and we won't have to worry about other tests in the platform being thrown off if they do a count of users in the `User` table or something like that.

Reply:
Python must hate me. For some reason my `@class method setupTestData` doesn't get called :(

Reply:
You're defining it like below, right? With a capital `U` in `Up`?
```python
    @classmethod
    def setUpTestData(cls):
        # Set up data for the whole TestCase
```

Reply:
I need to find a Homer Simpson 'DOH'! emoticon.